lib/Common/ConfigFlagsList.h

```diff
@@ -1477,6 +1487,9 @@ FLAGR(Number, JITServerIdleTimeout, "Idle timeout in seconds to do the cleanup i
 FLAGR(Number, JITServerMaxInactivePageAllocatorCount, "Max inactive page allocators to keep before schedule a cleanup", 10)
 
 FLAGNR(Boolean, StrictWriteBarrierCheck, "Check write barrier setting on none write barrier pages", false)
+FLAGNR(Boolean, WriteBarrierTest, "Always return true while checking barrier to test recycler regardless of annotation", DEFAULT_CONFIG_WriteBarrierTest)
+FLAGNR(Boolean, EnableHardwareWriteBarrier, "Use to turn off write watch to test software write barrier on windows", DEFAULT_CONFIG_EnableHardwareWriteBarrier)
```

Comment:
let me change GLOBAL_FORCE_USE_WRITE_BARRIER to GLOBAL_ENABLE_WRITE_BARRIER for compile time enabling the WriteBarrierPtr wrapper
and change EnableHardwareWriteBarrier to ForceSoftwareWriteBarrier for runtime forcibly use software write barrier even on windows 

---
In reply to: [87938342](https://github.com/Microsoft/ChakraCore/pull/1968#discussion_r87938342) [](ancestors = 87938342,87932027,87931407,87930847,87929333,87929125)